### ISSUE TITLE
Why the training process is abnormal?

### ISSUE THREAD

**Issue by junwenxiong** [wusonghe/TMFI-Net#2]
I tried to train the TMFI-Net from scratch. But the loss i got is abnormal. 
![image](https://github.com/wusonghe/TMFI-Net/assets/35716657/05fc57af-0280-4aec-8c0b-fb168960b7af)

Why this happend?

**Comment by wusonghe** [comment]
Hello

It is normal for the loss to fluctuate up and down during the short-term iteration process. If you output the loss of the entire training process, you will find that the loss has been decreasing.

In addition, the training data used in each epoch is randomly extracted from the data set. Because the data set is too large, it is not recommended to use all the data for training. For details, you can check the code in the dataloader.

good luck

**Comment by pomterre** [comment]
> I tried to train the TMFI-Net from scratch. But the loss i got is abnormal. ![image](https://private-user-images.githubusercontent.com/35716657/253806421-05fc57af-0280-4aec-8c0b-fb168960b7af.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQwMzE0MjEsIm5iZiI6MTcxNDAzMTEyMSwicGF0aCI6Ii8zNTcxNjY1Ny8yNTM4MDY0MjEtMDVmYzU3YWYtMDI4MC00YWVjLThjMGItZmIxNjg5NjBiN2FmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI1VDA3NDUyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg5NTU2ZWY0OGM4Mjk0ZTRkMTkyNTNhNmM4YTQ1Y2VhNjBmOWQyNjNiYTE2YjZkOWZkNmY0N2VlMWEzM2Y0OWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.aPryQosG4z_f1Xix90wRKknu186F-a5fLpuy9VCOyok)
> 
> Why this happend?

I'm getting the same issue when training from scratch with DHF1K training set. Have you solved this since?

**Comment by q-prashant** [comment]
hey @wusonghe can you kindly upload pre-trained weights to Google Drive or somewhere non-Chinese people can access it?
